aee.js

```diff
@@ -72,3 +72,13 @@ Array.prototype.count = function (spec) {
   }
   return result
 }
+
+Array.prototype.pluck = function (property) {
+  if (property === undefined) throw new ReferenceError('property is undefined')
+  const len = this.length
+  let result = []
+  for (let i = 0; i < len; i += 1) {
+    result.push(this[i] && this[i].hasOwnProperty(property) ? this[i][property] : null)
```

Comment:
Alright, I prefer to keep the already coded behavior, as a `null` value can be both understood as there was a missing object or the object did not contain the selected property while an `undefined` I think feels more like a failure in the function. Will add it to the documentation and merge it.